inspirehep/config.py

```diff
@@ -1437,7 +1438,7 @@ def _(x):
             },
             "source": {
                 "terms": {
-                    "field": "metadata.acquisition_source.method",
+                    "field": "metadata.acquisition_source.source",
```

Comment:
We might have at some other point sumbissions by email, or something.

The idea is to separate the source (user) from the method (form), the same way we separate them for harvests (source:arxiv,pos,wsp..., method:hepcrawl).

Or we can altogether remove one of the (method probably) and just use source.